### PR TITLE
Simplify_Jump_Host_Handling

### PR DESCRIPTION
**Pull request by esc**
Follow up url handlin from #2

### CODE REVIEW

**Comment by esc** [comment]
@schlomo please review and merge.

I also piggybacked an additional small fix within this PR.

File: src/main/scripts/cbas [hunk 1]
```diff
@@ -29,12 +29,10 @@ def get_jump_url(config):
...
-    if "://" in config.jump_host:
-        return config.jump_host
-
-    if config['password_provider'] == 'testing':
-        return 'http://' + config.jump_host
-    return 'https://' + config.jump_host
+    return (config.jump_host
+            if "://" in config.jump_host
+            else 'https://{0}'.format(config.jump_host)
+            )
```
Comment (schlomo) [inline src/main/scripts/cbas:32]: I think that the explicit if is easier to read than the inline if, at least in this specific case.

**Comment by esc** [comment]
@schlomo I addressed your comment, it has been fixed, please merge.

**Comment by schlomo** [comment]
Thanks a lot!

**Pull request merged**